runner/runners/invoke.go

```diff
@@ -369,17 +371,29 @@ func (inv *Invoker) run(cmd *runner.Command, id runner.RunID, abortCh chan struc
 			}).Info("Run timedout")
 		return runner.TimeoutStatus(id,
 			tags.LogTags{JobID: cmd.JobID, TaskID: cmd.TaskID, Tag: cmd.Tag})
+	case <-memCh:
```

Comment:
It would be very nice to write the memory cap here as well to save ourselves the inevitable trouble of telling people what it was. What if we set up the MemCh's to pass an execer.ProcessStatus and receive that here, and mix in the st.Error string into the formatted message?

Reply:
Good call, I like that. Will do